ludwig/models/gbm.py

```diff
@@ -28,6 +30,12 @@ def __init__(
         random_seed: int = None,
         **_kwargs,
     ):
+        if len(output_features) > 1:
+            raise ValueError("Only single task currently supported")
```

Comment:
For these kinds of errors, I always find it's helpful to include the failing state so the user knows what is wrong. Like:

```
Only single task currently supported, found: {feature_name_1: NUMBER, feature_name_2: CATEGORY}
```

And:

```
Model type GBM only supports number, category, or binary features, found: {feature_name: TEXT}
```
```